cmd/url-shortener/main.go

```diff
@@ -2,59 +2,61 @@ package main
 
 import (
 	"awesomeProject/internal/config"
+	"awesomeProject/internal/lib/logger/setup"
+	"awesomeProject/internal/lib/router"
 	"awesomeProject/internal/storage"
+	"context"
 	"fmt"
 	"log/slog"
+	"net/http"
 	"os"
 )
 
 const (
-	envLocal       = "local"
-	envDev         = "dev"
-	envProd        = "prod"
 	collectionName = "url-shortener"
+	dbName         = "mydb"
 )
 
 func main() {
-	//CONFIG
+	// CONTEXT
+	contextOur := context.TODO()
```

Comment:
ctx принято называть
```suggestion
	ctx := context.TODO()
```